pkg/sql/importer/exportcsv_test.go

```diff
@@ -451,11 +451,11 @@ func TestExportFeatureFlag(t *testing.T) {
 	sqlDB.Exec(t, `SET CLUSTER SETTING feature.export.enabled = FALSE`)
 	sqlDB.Exec(t, `CREATE TABLE feature_flags (a INT PRIMARY KEY)`)
 	sqlDB.ExpectErr(t, `feature EXPORT was disabled by the database administrator`,
-		`EXPORT INTO CSV 'nodelocal://0/%s/' FROM TABLE feature_flags`)
```

Comment:
I'm not sure how this was passing previously.. 

After my introduction of logging the url in`createPlanForExport` this test started failing w/ 
```
    exportcsv_test.go:458: error executing 'EXPORT INTO CSV 'nodelocal://0/%s/' FROM TABLE feature_flags': pq: parse "nodelocal://0/%s/": invalid URL escape "%s/"
    panic.go:522: -- test log scope end --
```

But note that in crdb if you try an export like this it fails with the same error anyways:
```
root@:26257/defaultdb> export into csv 'nodelocal://0/%s' from table foo;
ERROR: parse "nodelocal://0/%s": invalid URL escape "%s"
```

So i changed this `%s` to a normal path